ifrs17/DataModel/DataStructure.ipynb

```diff
@@ -1783,6 +1783,8 @@
                 "public record IfrsVariable : BaseDataRecord",
                 "\n{",
                 "\n    public double Value { get; init; }",
+                "\n    public double Value1 { get; init; }",
```

Comment:
I was wondering why we did this here, also in V7. The RawVar have double[], the Report Variable have Value and Projection as properties. Isn't there a way we can do that without hardcoding the number of projections?